apps/api_web/lib/api_web/params.ex

```diff
@@ -282,7 +289,13 @@ defmodule ApiWeb.Params do
         bad_filters = Map.keys(filter) -- keys
 
         if conn.assigns.api_version < "2019-04-05" or bad_filters == [] do
-          {:ok, Map.take(filter, keys)}
+          valid_filters =
+            filter
+            |> Map.take(keys)
+            |> Enum.reject(fn {k, v} -> k == "canonical" and v == "" end)
```

Comment:
suggestion: this doesn't seem like the right place for this filtering: I don't see anything else in here which knows about specific keys or values.